IntegrationTests.Common/Utilities/TestResultSerializer.cs

```diff
@@ -112,6 +112,8 @@ private void SerializeWithoutSpacing (object value)
     {
       if (value == null)
         _textWriter.WriteLine ("null");
+      else if (value is DateTime dateTime && dateTime.ToLongTimeString() == "00:00:00")
```

Comment:
Please add a comment here why this is needed.

Also, I think we should try to extract this change and the assiciated updates to the result-files into a dedicated commit (if that's not already the case). Assumption based on my experience: this is easy and quickly done.